templates/tips/board.html

```diff
@@ -0,0 +1,66 @@
+{% extends "base.html" %} {%load static%} {%block content%}
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Tips</title>
+    <style type="text/css">
+      .author,
+      .eom {
+        color: grey;
+      }
+      .grid{
+          height:100%;
+          width:100%;
+      }
+      .item {
+        width: 20%;
+        background-color: grey;
+        position:relative;
+      }
+      .item p {
+        color: white;
+        text-align: center;
+      }
+      .item * {
+        color: white;
+        text-align: center;
+      }
+      .item--width2 {
+        width: 50%;
+      }
+      .date{
+          position: absolute;
+          bottom:0;
+          right:0;
+      }
+      .edit{
+        position: absolute;
+        bottom:0;
+        left:0;
+      }
+    </style>
+    <script src="https://unpkg.com/packery@2/dist/packery.pkgd.min.js"></script>
+  </head>
+  <body>
+    <h1>Tips</h1>
+	<a class="btn btn-primary" href="add_tip/" role="button">Add new tip</a>
+    <div class="grid" >
+      {% for tip in tips %}
+      <div class="item">
+        <div class="category">{{tip.category}}</div>
+        <h6 class="author">author: {{ tip.author }}</h6>
+        <label class="date">date: {{ tip.date }}</label>
```

Comment:
Is it a last edited date? or a publish date?
I think it should mention whether it's the former or the latter

Reply:
It's always the publish date. Ill make it clearer

Reply:
fixed